dev-python/qtsass/qtsass-0.1.1.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Compile SASS files to Qt stylesheets "
+HOMEPAGE="https://github.com/spyder-ide/qtsass"
```

Comment:
Once again, there seems to be possibility for enabling doc and python tests, 
https://github.com/spyder-ide/qtsass/blob/master/setup.py#L55

Reply:
Fixed, I have removed the revbump of watchdog, but I can't do that for pathtools because pathtools is marked as stable. And repoman yells at me when I try to commit a ebuild with stable keywords or when I replace the stable keywords with unstable ones.

Reply:
You can just edit PYTHON_COMPAT without touching existing KEYWORDS, or having to revbump the ebuild. 
Was that a blocker to enabling python tests in this package? Python tests are usually the fastest way to determine whether the package works with latest python or not, so they are quite important. You should always enable them if they are available.

Also note that distutils-r1.eclass has a built-in function to enabling tests, *distutils_enable_tests*.